src/legacy/core_plugins/vis_default_editor/public/components/agg_group.test.tsx

```diff
@@ -55,6 +56,18 @@ describe('DefaultEditorAgg component', () => {
     setTouched = jest.fn();
     setValidity = jest.fn();
     reorderAggs = jest.fn();
+    schemas = new Schemas([
+      {
+        name: 'metrics',
+        group: 'metrics',
+        max: 1,
+      } as Schema,
+      {
+        name: 'buckets',
+        group: 'buckets',
+        max: 1,
+      } as Schema,
+    ]) as ISchemas;
```

Comment:
If https://github.com/elastic/kibana/pull/61410/files#r400751060 is done, this type cast also becomes unnecessary because it already has the right type.